README.md

```diff
@@ -30,36 +32,36 @@ Im Body wird eine Payload als JSON in folgendem Format verschickt:
 }
 ```
 
-Zu den Attributen im Detail:
+More details to the attributes:
 
 - `vorgangsnummer`:
 
-  Sowohl bei Änderungen im Vorgang als auch im Antrag wird in einer KEX-PUSH Message stets nur die Vorgangsnummer übermittelt.
+  For changes in either Vorgang or Antrag the KEX-PUSH message always only contains the Vorgangsnummer.
```

Comment:
```suggestion
  For changes of both Vorgang or Antrag the KEX-PUSH message contains only the Vorgangsnummer.
```

Reply:
```suggestion
  For changes of both Vorgang and Antrag the KEX-PUSH message only contains the Vorgangsnummer.
```

Reply:
jetzt ist das `and` irreführend: das klingt so, als ginge es darum, das Vorgang und Antrag _gleichzeitig_ geändert werden.

Reply:
jaja nicht nur deutsch ist schwierig 😅 

Ich würde es dann gerne so lassen :)

Reply:
```suggestion
  For changes in either Vorgang or Antrag the KEX-Push message always only contains the Vorgangsnummer.
```